File: src/main/java/HU/Tosad/table/Table.java
package HU.Tosad.table;

import java.util.ArrayList;
import java.util.List;

public class Table {
    private String name;
    private List<Table> tables;

    public Table(String name) {
        this.name = name;
    }

    public static List<Table> getAll() {
        List<Table> tables = new ArrayList<Table>();
        return tables;
    }
}

File: src/main/resources/static/script/table.js
$(document).ready(function() {
    fetch('http://localhost:8080/api/tosad/table', {method: 'GET'})
        .then((response) => {
            if (response.ok) {
                return response.json();
            }
        }).then((tables) => {
            let tableHtml = document.getElementById('table_table__body_data');
            let i;
            for (i = 0; i < tables.length; i++) {
                tableHtml.innerHTML += getTrHtml(tables[i].name);
            }
        });
});

function getTrHtml(tableName) {
    let html = '<a href="column.html?table=' + tableName + '" class="list-group-item list-group-item-action">' + tableName + '</a>';

    return html;
}